infra/libfuzzer-pipeline.groovy

```diff
@@ -65,6 +63,7 @@ def call(body) {
           writeFile file: "$wsPwd/${sanitizer}.rev", text: revText
           echo "revisions: $revText"
 
+          def dockerfile = "$workspace/oss-fuzz/$projectName/Dockerfile"
```

Comment:
Ah, didn't know that was a supported use case already. 

I've changed it so that dockerfile is still supported but optional.

For dicts/options, I can't think of a clean way of specifying where they exist if they're in an upstream repo and not oss-fuzz. In this case, maybe we'll just have to get them to copy to /out manually in build.sh